Cargo.toml

```diff
@@ -75,7 +80,10 @@ solana_rbpf = "=0.6.0"
 byteorder = "1.4"
 assert_cmd = "2.0"
 bincode = "1.3"
-ed25519-dalek = { version = "2", features = ["rand_core"] }
+# rand version 0.7 is needed for ed25519_dalek::keypair::generate, used in 
+# solana_tests/signature_verify.rs
+rand_07 = { package = "rand", version = "0.7" }
+ed25519-dalek = "1.0"
```

Comment:
Can you please add a comment explaining why we stick to `1.0` of this crate? Otherwise the next person bumping dependencies might not be aware or have forgotten about it